docs/source/download.rst

```diff
@@ -33,25 +33,20 @@ use Firedrake::
 
      source firedrake/bin/activate.csh
 
-Reporting installation bugs
----------------------------
-
-If ``firedrake-install`` fails to work, please report a bug so that we
-can fix it for you by creating a new `github issue
-<https://github.com/firedrakeproject/firedrake/issues>`__.  Please
-include the log file ``firedrake-install.log`` in your bug report.
-Similarly if ``firedrake-update`` fails, it produces a
-``firedrake-update.log`` file which will help us to debug the problem.
-
 Testing the installation
 ------------------------
 
-It is recommended to run the test suite after installation to check
-that the Firedrake installation is fully functional.  Activate the
-venv_ as above and then run::
+We recommend that you run the test suite after installation to check
+that Firedrake is fully functional. Activate the venv_ as above and
+then run::
 
   cd firedrake/src/firedrake
-  make alltest
+  make test_demos
+
+This will run all of the Firedrake demo codes, which exercise most of
+the functionality of the library. The demos should take about 10-20
+minutes to run. If these fail to run for any reason, please see the
+section below on how to diagnose and debug a failed installation.
```

Comment:
The only thing I'd consider changing here is giving the instructions on how to run all tests if they want to.

Reply:
If you really want I can put that in, but I don't think we should be suggesting that first-time users run a job that can take several hours. If there were a doc section on how to contribute to Firedrake then that would be the place to mention running `make alltest`.